### ISSUE TITLE
Bad Loss Of Model 

### ISSUE THREAD

**Issue by oggyfaker** [maudzung/SFA3D#29]
Hi author, 
I'm really appreciate you for the nice source code. But when i set up enviroment for training , i can't achieve good result like your repo . I got the nice loss of training , but in validation loss increase , i think may be the model is overfitting . May be can you recommand me some way to improve the accuracy ? 
Thanks ! 
 
![image](https://user-images.githubusercontent.com/44501897/107411052-a5e10a80-6b40-11eb-917e-92e77dfed7b6.png)

**Comment by vivi0604** [comment]
I have the same problem. Have you solved it?
